styleguide/template.md

```diff
@@ -102,32 +102,34 @@ Use for inline code, language keywords, NuGet package names, command-line comman
 To link to a header in the same Markdown file (also known as anchor links), you'll need to find out the id of the header you're trying to link to. To confirm the ID, view the source of the rendered article, find the id of the header (for example, `id="blockquote"`), and link using # + id (for example, `#blockquote`).
 The id is auto-generated based on the header text. So, for example, given a unique section named `## Step 2`, the id would look like this `id="step-2"`.
 
-- Example: [Chapter 1](#chapter-1)
+- Example: `[Chapter 1](#chapter-1)` produces [Chapter 1](#chapter-1).
 
 To link to a Markdown file in the same repo, use [relative links](https://www.w3.org/TR/WD-html40-970917/htmlweb.html#h-5.1.2), including the ".md" at the end of the filename.
 
-- Example: [Readme file](../readme.md)
-- Example: [Welcome to .NET](../docs/welcome.md)
+- Example: `[Readme file](../readme.md)` produces [Readme file](../readme.md).
+- Example: `[Welcome to .NET](../docs/welcome.md)` produces [Welcome to .NET](../docs/welcome.md).
 
 To link to a header in a Markdown file in the same repo, use relative linking + hashtag linking.
 
-- Example: [.NET Community](../docs/welcome.md#community)
+- Example: `[.NET Community](../docs/welcome.md#community)` produces [.NET Community](../docs/welcome.md#community).
```

Comment:
The relative links can be frustrating and error-prone for a deeply nested topic. (e.g., `../../../../standard/something.md`). I'm wondering if we should add the `~` for the root of the repo? (like `~/docs/standard/index.md`). The problem, though, is that since it's an extension, the link will 404, so that may be a bad idea.

Reply:
I added language here based on the practices we **usually** follow. I don't know that we have a clear rule here.

/cc @mairaw if she wants to add her thoughts.